features/networking/multus.feature

```diff
@@ -1394,4 +1394,58 @@ Feature: Multus-CNI related scenarios
       | <%= project.name %>:macvlan-bridge-whereabouts-pod1:<%= cb.pod_uid %> |
       | <%= cb.pod_eth0_ip %>                                                 |
       | <%= cb.pod_net1_ip %>                                                 |
+      
+  # @author weliang@redhat.com
+  # @case_id OCP-31999
+  @admin
+  Scenario: Whereabouts with exclude IP address	
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    Given I obtain test data file "networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-excludeIP.yaml"
+    When I run oc create as admin over "whereabouts-excludeIP.yaml" replacing paths:
+      | ["metadata"]["namespace"] | <%= project.name %>          |
```

Comment:
Please reduces the whitespaces in the end

Reply:
@anuragthehatter Fixed, thanks!